INSTALL.md

```diff
@@ -0,0 +1,166 @@
+	- [Ubuntu and Debian](#ubuntu-and-debian)
+- [Install from source](#install-from-source)
+	- [Build tools and dependencies](#build-tools-and-dependencies)
+		- [Debian/Ubuntu](#debianubuntu)
+		- [Other Linux distributions](#other-linux-distributions)
+		- [MacOS X](#macos-x)
+		- [Windows](#windows)
+	- [Quick start](#quick-start)
+	- [Download the sources](#download-the-sources)
+		- [Download source archive](#download-source-archive)
+		- [Clone from GitHub](#clone-from-github)
+	- [3. Build and Install](#3-build-and-install)
+- [Getting Started](#getting-started)
+- [Cross Compiling Orocos](#cross-compiling-orocos)
+
+<!-- /TOC -->
+
+## Introduction
+
+This sections explains the supported Orocos targets and the Orocos versioning scheme.
+
+### Supported platforms (targets)
+
+Orocos was designed with portability in mind. Currently, we support RTAI/LXRT (http://www.rtai.org), GNU/Linux userspace, Xenomai (http://www.xenomai.org), Mac OS X (http://www.apple.com) and native Windows using Microsoft Visual Studio. So, you can first write your software as a normal Linux/Mac OS X program, using the framework for testing and debugging purposes in plain userspace (Linux/Mac OS X) and recompile later to a real-time target or MS Windows.
+
+### The versioning scheme
+
+A particular version is represented by three numbers separated by dots. For example:
+
+2.8.1 : Release 2, Feature update 8, bug-fix revision 1.
+
+### Dependencies on other libraries
+
+Before you install Orocos, verify that you have the following software installed on your platform:
+
+| Program / Library               | Minimum Version               | Description                                                                                                                                                                                                                                                                            |
+|---------------------------------|-------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CMake                           | 2.8.3 (all platforms)         | See resources on cmake.org for pre-compiled packages in case your distribution does not support this version                                                                                                                                                                           |
+| Boost C++ Library               | 1.33.0 (1.40.0 recommended!)  | Boost.org from version 1.33.0 on has a very efficient (time/space) lock-free smart pointer implementation which is used by Orocos. 1.36.0 has boost::intrusive which we require on Windows with MSVS. 1.40.0 has a shared_ptr implementation we require when building Service objects. |
```

Comment:
Update minimum Boost version to 1.40, check https://github.com/orocos-toolchain/rtt/pull/199.